main.go

```diff
@@ -91,7 +92,7 @@ func main() {
 
 	s := &http.Server{
 		// Set the addr to localhost so that we only listen locally.
-		Addr:    fmt.Sprintf("localhost:%d", *port),
+		Addr:    fmt.Sprintf("%s:%d", *location, *port),
```

Comment:
...so here, we should probably do something like:

```
Addr: net.JoinHostPort(*location, strconv(*port)),
```

That'll make sure that it gets enclosed in square brackets, if it's an IPv6 address.